helm/values.yaml

```diff
@@ -198,6 +312,12 @@ daskhub:
           - name: driven-data
             mountPath: /driven-data/
 
+      extraFiles:
+        spark_default_configuration:
+          mountPath: /home/jovyan/.ipython/profile_default/startup/00-spark-conf.py
+        spark_executor_template:
+          mountPath: /home/jovyan/executor-template.yml
```

Comment:
It seems that these mountPaths may conflict with the `/home/jovyan` vpc; I'll add a fix shortly. The idea is to mount files into its own unique location and to copy them over in the post_start hook.

Reply:
e88379e